lib/AvirWrapper.php

```diff
@@ -21,10 +21,12 @@
 
 
 class AvirWrapper extends Wrapper{
-	
+
+	const AV_EXCEPTION_MESSAGE = 'Antivirus app is either misconfigured or inaccessible. %s';
```

Comment:
@settermjd  Antivirus app is meant to be ownCloud app. It could be misconfigured but can't be inaccessible.
Inaccessible is external antivirus: server/port  or socket. E.g. when clamav service is accidentally stopped.